packages/react-docs/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "./lib/",
+        "jsx": "react",
+        "baseUrl": "./",
+        "strictNullChecks": false,
```

Comment:
It requires a lot of work to fix that in the whole website package, but let's please use strict null checks in this one. It's super usefull. Let's be strict.

Reply:
Agreed, this should be a subsequent PR. It's actually a non-trivial amount of work since we do heavily rely on it in these components :/